packaging/wheel/relocate.py

```diff
@@ -79,7 +79,7 @@ def rehash(path, blocksize=1 << 20):
     h = hashlib.sha256()
     length = 0
     with open(path, "rb") as f:
-        for block in read_chunks(f, size=blocksize):
```

Comment:
We could delete the `read_chunks` function as it's no longer used, but I wasn't sure if it was part of the public API or not.

Reply:
It is not. Go ahead and delete it.